authui/src/authflowv2/customSelect.ts

```diff
@@ -147,11 +149,10 @@ export class CustomSelectController extends Controller {
   }
 
   close() {
-    if (this.dropdownTarget.classList.contains("hidden")) return;
+    if (this.dropdownContainerTarget.classList.contains("hidden")) return;
 
-    this.dropdownTarget.classList.add("hidden");
+    this.dropdownContainerTarget.classList.add("hidden");
     this.triggerTarget.setAttribute("aria-expanded", "false");
-    this.triggerTarget.focus();
```

Comment:
@IniZio 

Any reason for us to focus the trigger target when closed the dropdown?

Reply:
Wanted to refocus the trigger after user just selected an item in the dropdown.

Reply:
I know but what is it for? The reason I tried to remove this focus action is because when I click to phone input box to close the dropdown, it immediate blurs the phone input because here called to focus the trigger.

Reply:
But html select should have similar behaviour? 

If you have an html select and input in same form, clicking the input with select opened will not focus the input immediately.